### DISCUSSION TITLE
PDE Point Resampling for Anchor points

### DISCUSSION THREAD

**Discussion by shanmukhi1998** [lululxvi/deepxde#1349] [General]
Hello all

I have a query. I am solving an inverse problem where I am solving a system of PDEs. I have num_domain = 0 and anchors set as the datapoints that I have (~8000). Is there a provision to resample say, 1000 anchor points every 1000 iterations. I would really appreciate any help in this regard.

Thanks.

**Comment by lululxvi** [comment]
Resampling anchors is not supported now.
